package.json

```diff
@@ -19,6 +19,13 @@
     "release:publish": "./scripts/publishRelease.sh",
     "test": "jest --runInBand"
   },
+  "dependencies": {
+    "classnames": "^2.0.0",
+    "invariant": "^2.0.0",
+    "lodash": "^4.0.0",
+    "tslib": "^2.2.0",
+    "uniforms": "^3.5.2"
```

Comment:
Could you remove those? they're duplicated from the `devDependencies`.

Reply:
I'll remove them from the devDeps. I'm trying to align how this is built to how they build the thing upstream, see [here](https://github.com/vazco/uniforms/blob/master/packages/uniforms-semantic/package.json#L37-L40)